en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -497,8 +497,9 @@ load fields off of contained associations, you can use ``autoFields()``::
 
     // Select id & title from articles, but all fields off of Users.
     $query->select(['id', 'title'])
-        ->contain(['Users'])
-        ->autoFields(true);
+        ->contain(['Users' => function($q) {
+			return $q->autoFields(true);
+		          }]);
```

Comment:
@markstory I can confirm that if you do ``autoFields()`` the way it is done in the doc, you get all fields for the ``articles`` table :

With this Query : 
```php
$query = $this->Articles->find()->select(['id', 'title'])
            ->contain(['Users'])
            ->autoFields(true);
```

I get : 
```
object(App\Model\Entity\Article) {
	'id' => (int) 1,
	'title' => 'first',
	'content' => 'plop',
	'user_id' => (int) 1,
	'created' => null,
	'modified' => null,
	'user' => object(App\Model\Entity\User) {
		'id' => (int) 1,
		'username' => 'bob',
		'created' => null,
		'modified' => null,
		//[...]
	},
	//[...]
}
```

My schema is a simple ``articles`` belongsTo ``users`` (it also works the other way around).
The change @caBBAlainB did makes this work.
I'll try to see if this is working for all types of associations.